test/dart/leak_tracking/_object_tracker_test.dart

```diff
@@ -223,6 +251,15 @@ void main() {
   });
 
   group('$ObjectTracker with stack traces', () {
+    late _MockFinalizerBuilder finalizerBuilder;
+    late _MockGcCounter gcCounter;
+    late ObjectTracker tracker;
+
+    /// Emulates GC.
+    void gc(Object object) {
+      finalizerBuilder.finalizer.finalize(identityHashCode(object));
+    }
```

Comment:
nit: since this is duplicated, what do you think about pulling this outside of the groups for shared use, and passing the `finalizerBuilder` as a parameter?